applications/fishing-map/src/features/workspace/workspace.mock.ts

```diff
@@ -657,31 +658,41 @@ const workspace: Workspace = {
       ],
     },
   ],
+  // We need to "instanciate" every dataview we want to use in the workspace
+  // think as same model than mapbox-gl with source (dataview) and layer (dataviewConfig)
```

Comment:
I strongly think this should be called `dataviewsInstances`

Reply:
ok, let's do it! but as this is something that becomes with libraries and API updates I'll do it in another PR